pyrovider/tools/dicttools.py

```diff
@@ -12,7 +12,7 @@ def dictpath(dictionary: dict, path: list):
 
 def dictiter(arg):
     if isinstance(arg, dict):
-        return iter(arg.items())
+        return iter(list(arg.items()))
```

Comment:
`arg.items` is already an iterator, remove both `iter` and `list` castings.

Reply:
`2to3` :sigh: